Mid.html

```diff
@@ -0,0 +1,108 @@
+<!Doctype html>
+<html>
+    <head lan="en">
+        <link rel="stylesheet" type="text/css" href="Mid.css">
+        <title>MiD</title>
+    </head>
+    <body>
+	<div class="resume">
+         <div class = "externalDiv">
+      <div class = "internalDiv">
```

Comment:
Why 3 divs

Reply:
The outermost div contains the whole resume, other two divs are for header, which contains the heading and the sub-heading.